tokens/color.json

```diff
@@ -99,6 +99,36 @@
           "themeable": true
         }
       },
+      "teal": {
```

Comment:
I'd recommend to change all hex colors to lowercase to stick to the convention. The PhpStorm [String Manipulation plugin](https://plugins.jetbrains.com/plugin/2162-string-manipulation) can assist greatly with that.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/105/commits/5786200cc1008821435b33004379fb7322f707d6